src/App.jsx

```diff
@@ -1,13 +1,196 @@
+                        value={formState.firstName}
+                        updateCounter={handleChange}
+                    />
+                    <Input
+                        inputID="lastName"
+                        inputType="text"
+                        inputLabel="Achternaam"
+                        value={formState.lastName}
+                        updateCounter={handleChange}
+                    />
+                    <Input
+                        inputID="age"
+                        inputType="number"
+                        inputLabel="Leeftijd"
+                        value={formState.age}
+                        updateCounter={handleChange}
+                    />
+                    <Input
+                        inputID="zipCode"
+                        inputType="text"
+                        inputLabel="Postcode"
+                        value={formState.zipCode}
+                        updateCounter={handleChange}
+                    />
+                    <label htmlFor="frequency">Bezorgfrequentie
+                        <select name="frequency" value={formState.frequency} onChange={handleChange} className="input">
+                            <option value="iedere week">Iedere week</option>
+                            <option value="om de week">Om de week</option>
+                            <option value="iedere maand">Iedere maand</option>
+                        </select>
+                    </label>
+                    <div className="radioWrapper">
+                        <label htmlFor="time-day">
+                            <input type="radio" id="time-day" value="day" name="time" checked={formState.time === "day"}
+                               onChange={handleChange}/>Overdag
+                        </label>
+                        <label htmlFor="time-night">
+                            <input type="radio" id="time-night" value="night" name="time"
+                               checked={formState.time === "night"}
+                               onChange={handleChange}/>'s avonds
+                        </label>
+                    </div>
+                    <label htmlFor="comments">Opmerkingen
+                    <textarea
+                        id="comments"
+                        name="comments"
+                        value={formState.comments}
+                        onChange={handleChange}
+                        placeholder="Enter your comments here..."
+                    />
+                </label>
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+                    <label htmlFor="termsConditions">
+                        <input type="checkbox" id="termsConditions" name="termsConditions"
+                               value={formState.termsConditions}
+                               onChange={handleChange}/>Ik ga akkoord met de voorwaarden</label>
+                    <Button
+                        buttonType="button"
```

Comment:
Het type van een button in een form element zou een "submit" moeten zijn en geen "button".